be/src/column/column_helper.cpp

```diff
@@ -240,6 +240,19 @@ bool ColumnHelper::is_all_const(const Columns& columns) {
     return std::all_of(std::begin(columns), std::end(columns), [](const ColumnPtr& col) { return col->is_constant(); });
 }
 
+pair<bool, size_t> ColumnHelper::num_packed_rows(const Columns& columns) {
```

Comment:
We needn't this method.   we could always use columns[0] size to get the chunk row number.

Reply:
I think this method is very useful and could solve many problems.